packages/components/src/internal/actions.ts

```diff
@@ -331,7 +331,17 @@ function buildQueryParams(
     containerFilter?: Query.ContainerFilter
 ): Record<string, any> {
     const filters: Record<string, any> = filterArray.reduce((_filters, filter) => {
-        _filters[filter.getURLParameterName()] = filter.getURLParameterValue();
+        const name = filter.getURLParameterName();
+        let value = filter.getURLParameterValue();
+        if (_filters[name] !== undefined) {
+            let values = _filters[name];
+            if (!Array.isArray(values)) {
+                values = [values];
+            }
+            values.push(value);
+            value = values;
+        }
+        _filters[name] = value;
```

Comment:
and this change works because the getURLParameterName() here includes both the field name and the operator name, right?

Reply:
ah ok, yeah these are the "query.*~inexpdescendantsof" type filters, so that makes sense why this issue was happening and this fixes it.